Formula/cpr.rb

```diff
@@ -16,14 +16,15 @@ class Cpr < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "openssl@1.1"
+  depends_on "openssl@3"
 
   uses_from_macos "curl"
 
   def install
     args = std_cmake_args + %w[
```

Comment:
```suggestion
    args = std_cmake_args + %W[
```
You need `%W` for interpolation.